app/master/build_request.py

```diff
@@ -22,6 +22,14 @@ class BuildRequest(object):
         "job_name": "clusterrunner_configured_job",
```

Comment:
Ah yeah, that's what I meant. Maybe outside scope for this specific change then.